backend/src/main/java/com/woowacourse/levellog/member/application/MemberService.java

```diff
@@ -71,6 +75,17 @@ private void checkSameGithubId(final MemberCreateDto request) {
         }
     }
 
+    private Member createMember(final MemberCreateDto request) {
+        final Member member = request.toEntity();
+        final Optional<Nickname> nickname = nicknameRepository.findByGithubNickname(
+                request.getNickname());
```

Comment:
```suggestion
        final Optional<Nickname> nickname = nicknameRepository.findByGithubNickname(request.getNickname());
```
그렇게 가독성에 크게 도움이 되는거 같지 않아서 붙이는걸 선호합니당 👍